src/Honeybee_Honeybee.py

```diff
@@ -5022,7 +5025,116 @@ def assignScheduleBasedOnProgram(self, component = None):
         
         # find all the patameters and assign them to 
         self.isSchedulesAssigned = True
+        
+    def getConstructionsByStandardClimateZone(self,standard,climateZone,component=None):
+        
+        openStudioStandardLib = sc.sticky["honeybee_OpenStudioStandardsFile"]
+        
+        try:
+            schedulesAndLoads = openStudioStandardLib['space_types']['90.1-2007']['ClimateZone 1-8'][self.bldgProgram][self.zoneProgram]
+            
+        except:
```

Comment:
@mostaphaRoudsari 

See the new code I've written here https://github.com/antonszilasi/Honeybee/blob/master/src/Honeybee_Honeybee.py#L5033-L5066

to address this issue